scripts/generate-inline-script-hash.js

```diff
@@ -0,0 +1,29 @@
+var sjcl = require("../src/assets/scripts/sjcl.js");
```

Comment:
This is a rather specialized script. I think it would be best to add a comment block at the top of this file explaining exactly what happens and why this is needed.

Reply:
Ah alright, added comment to the top in the next commit